crates/holochain_util/src/jsonschema.rs

```diff
@@ -0,0 +1,13 @@
+use schemars::{
```

Comment:
Module doc comment? Can be one line, just quickly describing what this module is for